packages/web-app/src/config/views.js

```diff
@@ -1,5 +1,7 @@
 const views = {
   appealAlreadyReviewed: 'appeal-already-reviewed',
+  questionnaireAlreadyReviewed: 'questionnaire-already-reviewed',
+  questionnaireForReview: 'questionnaires-for-review',
```

Comment:
@demzzydev I believe this file is used to define the views name. _questionnaireForReview_ does not seem to be a view